.storybook/formDecorator.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import { Button } from '@dhis2/ui-core'
+
+import { Form, FormSpy } from '../src'
+
+const formProps = {
+    onSubmit: values => {
+        console.log(
+            '++++++++++++++++\n',
+            'Form was submitted with values:\n',
+            values,
+            '\n----------------'
+        )
+    },
+    mutators: {},
+}
+
+class FormWithSpyAndSubmit extends React.Component {
+    state = {
+        cypressProps: {},
+    }
+
+    componentDidMount() {
+        window.updateCypressProps = this.updateCypressProps
+        window.clearCypressProps = this.clearCypressProps
+    }
+
+    componentWillUnmount() {
+        delete window.updateCypressProps
+        delete window.clearCypressProps
+    }
+
+    updateCypressProps = updateObj => {
+        const cypressProps = {
+            ...this.state.cypressProps,
+            ...updateObj,
+        }
+        this.setState({ cypressProps })
+    }
+
+    clearCypressProps = () => {
+        this.setState({ cypresProps: {} })
```

Comment:
Typo: cypresProps

Reply:
This was resolved, but it seems like the typo is still in there. Or is this intentional?

Reply:
Nevermind, this was resolved, apparently clicking through on the filename takes you to the original version, not the latest one.